simple-tubs-letter:0.1.2

Cangarw opened the pull request:
<!--
Thanks for submitting a package! Please read and follow the submission guidelines detailed in the repository's README and check the boxes below. Please name your PR as `name:version` of the submitted package.

If you want to make a PR for something other than a package submission, just delete all this and make a plain PR.
-->

I am submitting
- [ ] a new package
- [x] an update for a package

<!--
Please add a brief description of your package below and explain why you think it is useful to others. If this is an update, please briefly say what changed.
-->

The user can now choose their font by defining it in the metadata.typ. This is necessary for Windows user, as the standard font of this package is missing for a standard installation.

<!--
The following box only needs to be checked for **template** submissions. If you're submitting a package that isn't a template, you can delete the following section. See the guidelines section about licenses in the README for more details.
-->
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

elegaanz commented:
Thank you for this update. :)

For info, it is possible to give multiple fonts to `text.font` (if the first one is not found, it will try the next in the array, and so on), maybe you could provide a default that works on Linux and fallback to a similar looking Windows font? But letting the user customize their font is a nice feature to have as well, I'm just suggesting that in case you want to provide better defaults that would work for everyone without having to edit `metadata.typ`.

Also, before merging, could you please update the import in `letter.typ` to 0.1.2?

Cangarw commented:
I fixed the old version number in the `letter.typ` and added an array of fonts in the template. Should be good to go now. 

Thanks again for the suggestion!

elegaanz commented:
Thanks!
